warsow.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>warsow.desktop</id>
+  <name>Warsow</name>
+  <summary>Fast paced 3D first person shooter</summary>
+  <description>
+    <p>Warsow is set in a futuristic cartoonish world where rocketlauncher-wielding pigs and lasergun-carrying cyberpunks roam the streets. It is a completely free, fast-paced first-person shooter for Windows, Linux, and Mac OS X.</p>
+    <p>Speed and movement is what Warsow is all about. Like a true cyberathlete you jump, dash, dodge, and walljump your way through the game. Grab power-ups before your enemy does, plant a bomb before anyone sees you, and steal the enemy’s flag before they know what is going on!</p>
+    <p>Our goal is to offer a fast and fun competitive first-person shooter without hard graphical violence - Warsow has no blood or guts flying around. Red stars instead of blood indicate hits and colored cubes replace guts as gib effects.</p>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+ AND CC-BY-SA-4.0 AND CC-BY-ND-4.0</project_license>
```

Comment:
this shouldn't include the license of the assets.

Reply:
Why? They are copyrighted as well!

Reply:
Compare also https://flathub.org/apps/details/com.realm667.Wolfenstein_Blade_of_Agony